src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/ReferenceDiploidExactAFCalculator.java

```diff
@@ -209,12 +211,16 @@ private static void pushData(final ExactACset targetSet,
                                  final List<double[]> genotypeLikelihoods) {
         final int totalK = targetSet.getACsum();
 
-        for ( int j = 1; j < targetSet.getLog10Likelihoods().length; j++ ) {
-            if ( totalK <= 2*j ) { // skip impossible conformations
+        final double[] targetSetLog10Likelihoods = targetSet.getLog10Likelihoods();
+        final double[] dependentSetLog10Likelihoods = dependentSet.getLog10Likelihoods();
+        final int[] counts = targetSet.getACcounts().getCounts();
+
+        for ( int j = (totalK/2)-1, n = targetSetLog10Likelihoods.length; j < n; j++ ) {
```

Comment:
good catch. i think totalK may be 0 sometimes. I'll leave this as `for ( int j = 1, n = targetSetLog10Likelihoods.length; j < n; j++ ) {`